jbehave-support-core/src/main/java/org/jbehavesupport/core/splunk/SplunkSearchContext.java

```diff
@@ -0,0 +1,13 @@
+package org.jbehavesupport.core.splunk;
```

Comment:
Minor, but I would probably put it in internal? (I know that right now rest is not there and sql is - we should probably align that in the future - one way or the other 🙂)

Reply:
Ok it went to internal